api/client/node/update.go

```diff
@@ -47,7 +51,6 @@ func runUpdate(dockerCli *client.DockerCli, nodeID string, mergeNode func(node *
 		return err
 	}
 
-	fmt.Fprintf(dockerCli.Out(), "%s\n", nodeID)
```

Comment:
Why this change? `args[0]` is already passed as nodeID, so this message would be the same, and I think it makes more sense to have this in `runUpdate()`